phoebe/backend/universe.py

```diff
@@ -1998,7 +1998,9 @@ def is_convex(self):
 
     @property
     def needs_recompute_instantaneous(self):
-        return self.needs_remesh
+        # recompute instantaneous for asynchronous spots, even if meshing
+        # doesn't need to be recomputed
```

Comment:
while this plugs the leak, would it not be better to handle async spots at the teff level rather than recomputing the entire mesh?

Reply:
yes, agreed, I created a follow-up ticket to revisit this entire remeshing/computing infrastructure so we can remain as efficient as possible without breaking any assumptions.